src/components/progress-report/workflow/transitions.ts

```diff
@@ -17,36 +17,71 @@ export const Transitions = defineTransitions({
     to: Status.InProgress,
     label: 'Start',
     type: Type.Approve,
-    notify: 'FPM',
+    notify: {
+      TeamMembersWithRoles: [
+        Role.ProjectManager,
+        Role.RegionalDirector,
+        Role.FieldOperationsDirector,
+      ],
```

Comment:
If every transition is going to have these recipients I would just implicitly apply them in the service instead of copying them for every transition here.